test/c3/appframework/appframework_test.go

```diff
@@ -758,7 +728,7 @@ var _ = Describe("c3appfw test", func() {
 
 			// Verify Apps are downloaded by init-container
```

Comment:
Nit: We should remove the init-container strings from comments as we no longer use them

Reply:
This is addressed in the PR for CSPL-1414, the next one to go in.